session/handler.go

```diff
@@ -349,6 +367,83 @@ func (h *Handler) adminListSessions(w http.ResponseWriter, r *http.Request, ps h
 	h.r.Writer().Write(w, r, sess)
 }
 
+// Session Get Request
+//
+// The request object for getting a session in an administrative context.
+//
+// swagger:parameters adminGetSession
+// nolint:deadcode,unused
+type adminGetSessionRequest struct {
+	// ExpandOptions is a query parameter encoded list of all properties that must be expanded in the Session.
+	// Example - ?expand=Identity&expand=Devices
+	// If no value is provided, the expandable properties are skipped.
+	//
+	// enum: Identity,Devices
```

Comment:
Wow! I did not know this works! Ping @zepatrik !

```suggestion
	// enum: identity,devices
```

Reply:
<img width="911" alt="Screenshot 2022-11-04 at 09 53 13" src="https://user-images.githubusercontent.com/6560194/199931973-bf8912d2-92aa-4c78-90d2-65d7bf9043ae.png">

Yes, it does put this out in the spec like in the screenshot. But it still doesn't work when I try to get this to link to the definition. So ```
ExpandOptions []Expandable `json:"expand"` ``` would be the right thing to do imo to restrict the list options to the enum ones.

Reply:
I see, you can take a look into our internal docs to see how we usually solve enums: https://intranet.ory.sh/docs/dev/api-guidelines#enums